sql/report_queries/missing_items/missing_items.sql

```diff
@@ -0,0 +1,140 @@
+/* Change the lines below to adjust the date and location filters */
+WITH parameters AS (
+    SELECT
+        '2000-01-01'::date AS start_date,
+        '2022-01-01'::date AS end_date,
+        'Missing'::varchar AS item_status_filter, --  Should be 'Missing'
+              ---- Fill out one location or service point filter, leave others blank ----
+        ''::varchar AS item_permanent_location_filter, -- 'Main Library'
+        ''::varchar AS item_temporary_location_filter, -- 'Annex'
+        ''::varchar AS holdings_permanent_location_filter, -- 'Main Library'
+        ''::varchar AS holdings_temporary_location_filter, -- 'Main Library'
+        ''::varchar AS effective_location_filter -- 'Main Library'
+),
+---------- SUB-QUERIES/TABLES ----------
+item_subset AS (
+    SELECT 
+        item_id,
+        holdings_record_id,
+        status_name AS item_status,
+        status_date::timestamp AS item_status_date,
+        barcode,
+        call_number,
+        enumeration,
+        chronology,
+        copy_number,
+        volume
+    FROM 
+        folio_reporting.item_ext AS it
+    WHERE
+        status_date::timestamp >= (SELECT start_date FROM parameters)
+    AND status_date::timestamp < (SELECT end_date FROM parameters)
+    AND (status_name = (SELECT item_status_filter FROM parameters)
+        OR (SELECT item_status_filter FROM parameters) = '')
+),
+ranked_loans AS (
+    SELECT
+        cl.item_id,
+        cl.id AS loan_id,
+        cl.return_date AS loan_return_date,
+        cl.item_status,
+        rank() OVER (PARTITION BY cl.item_id ORDER BY cl.return_date DESC) AS return_date_ranked
+    FROM
+        public.circulation_loans AS cl
+        RIGHT JOIN item_subset AS its ON cl.item_id = its.item_id
+),
+latest_loan AS (
+    SELECT
+        item_id,
+        loan_id,
+        item_status,
+        loan_return_date 
+    FROM
+        ranked_loans
+    WHERE
+        ranked_loans.return_date_ranked = 1
+),
+item_notes_list AS (
+    SELECT
+        itn.item_id,
+        string_agg(DISTINCT itn.note, '|'::text) AS notes_list
+    FROM
+        folio_reporting.item_notes AS itn
+        RIGHT JOIN item_subset AS its ON itn.item_id = its.item_id
+    GROUP BY
+        itn.item_id
+),
+instance_subset AS (
+    SELECT 
+        ie.instance_id 
+    FROM 
+        item_subset AS its
+        LEFT JOIN folio_reporting.holdings_ext AS he ON its.holdings_record_id = he.holdings_id
+        LEFT JOIN folio_reporting.instance_ext AS ie ON he.instance_id = ie.instance_id
+),
+publication_dates_list AS (
+    SELECT
+        ip.instance_id,
+        string_agg(DISTINCT date_of_publication, '|'::text) AS publication_dates_list
+    FROM
+        folio_reporting.instance_publication AS ip
+        RIGHT JOIN instance_subset AS ins ON ip.instance_id = ins.instance_id
+    GROUP BY
+        ip.instance_id
+)
+    ---------- MAIN QUERY ----------
+SELECT
+    (SELECT start_date::varchar FROM parameters) || 
+        ' to '::varchar || 
```

Comment:
::varchar is not needed for inline string literals

Reply:
removed in latest commit